cantera/meta.yaml

```diff
@@ -116,15 +117,16 @@ outputs:
     test:
       requires:
         - pytest
-      imports:
-        - cantera
```

Comment:
I don't see a reason to remove this? It's just a very basic check that the package can be imported